symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/health/HealthService.java

```diff
@@ -0,0 +1,95 @@
+package com.symphony.bdk.core.service.health;
+
+import com.symphony.bdk.core.auth.AuthSession;
+import com.symphony.bdk.gen.api.SignalsApi;
+import com.symphony.bdk.gen.api.SystemApi;
+import com.symphony.bdk.gen.api.model.AgentInfo;
+import com.symphony.bdk.gen.api.model.V2HealthCheckResponse;
+import com.symphony.bdk.gen.api.model.V3Health;
+import com.symphony.bdk.http.api.ApiException;
+import com.symphony.bdk.http.api.ApiRuntimeException;
+
+import org.apiguardian.api.API;
+
+/**
+ * Service class for checking health of the Agent server.
+ */
+@API(status = API.Status.STABLE)
+public class HealthService {
+
+  private final SystemApi systemApi;
+  private final SignalsApi signalsApi;
```

Comment:
do we intend to fix the swagger spec to fix that strange thing ?

Reply:
we should open a PLAT ticket to fix this thing. I can open if it's really needed